gazebo/PKGBUILD

```diff
@@ -47,14 +47,14 @@ build() {
   mkdir -p build && cd build
 
   export PKG_CONFIG_PATH=/usr/lib/ffmpeg4.4/pkgconfig
-  cmake .. -GNinja \
+  cmake .. -G "Unix Makefiles" \
```

Comment:
``Unix Makefiles`` is the default generator, so you can just remove this option.

Reply:
Thanks for the comments. I updated the PKGBUILD.